packages/x/templates/blueprint/partials/formXElement.tpl

```diff
@@ -112,9 +112,7 @@
             >
               {options.map((option) => (
                 <Ant.Select.Option value={option.value} key={option.value}>
-                  {t(`{{ title }}_enums.${option.label.toLowerCase()}`)
-                    ? t(`{{ title }}_enums.${option.label.toLowerCase()}`)
-                    : option.label}
+                  {t(`{{ title }}_enums.${option.label.toLowerCase()}`) ?? option.label}
```

Comment:
Maybe we should do this at translator level, allow a defaul when no value is found, this was present in other frameworks I've worked with .translate(translateString, defaultValue?)